Laravel Blade.sublime-syntax

```diff
@@ -4,7 +4,7 @@ name: Laravel Blade
 file_extensions:
   - blade
   - blade.php
-scope: text.blade
+scope: source.blade
```

Comment:
@jbrooksuk Let me investigate that ;-) Looking at [the unoficial doc](http://docs.sublimetext.info/en/latest/extensibility/syntaxdefs.html) they clearly state that for languages we should use scope, so there has to be a way. Maybe by adding other scopes to the list (ie: text.source.blade, but probably not).

That being said, I never knew that you could expand classes with div.foo, that's awesome!